dashboard/app/models/peer_review.rb

```diff
@@ -113,6 +113,19 @@ def mark_user_level
       return
     end
 
+    if escalated? && user_level.best_result == Activity::UNREVIEWED_SUBMISSION_RESULT && !from_instructor
+      # If this has been escalated, create a review for an instructor to review
+      PeerReview.find_or_create_by(
+        submitter: submitter,
+        reviewer: nil,
+        script: script,
+        level: level,
+        level_source_id: level_source_id,
+        status: 2
+      )
```

Comment:
Should this and [this](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/peer_review.rb#L142-L144) be combined / encapsulated into a method?